opencog/moses/deme/deme_expander.cc

```diff
@@ -489,8 +490,10 @@ void deme_expander::optimize_demes(int max_evals, time_t max_time)
 
 			// Optimize
 			if (_params.atomspace_port) {
+				string_seq input_labels ;
+				for(int i=1; i< _labels.size(); i++) { input_labels.push_back(_labels.at(i));}
 				ComboToAtomese to_atomese(_output_type);
-				atomese_based_scorer cpx_scorer = atomese_based_scorer(_cscorer, _reps[i], _params.reduce_all, to_atomese, _output_type,
+				atomese_based_scorer cpx_scorer = atomese_based_scorer(_cscorer, _reps[i], _params.reduce_all, to_atomese,input_labels,
```

Comment:
Minor formatting remark, there's a space after `,` as expressed in the opencog dev standard, see last item of https://wiki.opencog.org/w/Development_standards#Formatting